### PR TITLE
(DOCSP-33221): "Release notes, redirects, source constant updates for v1.11.0 and v1.12.0.

### PR DESCRIPTION
**Pull request by corryroot**
@sarahsimpers, I added the release notes for v1.11.0 and 1.12.0 and checked out both submodules.

[STAGE](https://preview-mongodbcorryroot.gatsbyjs.io/atlas-cli/DOCSP-33221/atlas-cli-changelog/)

[DOCSP-33221](https://jira.mongodb.org/browse/DOCSP-33221)

[DOCSP-33079](https://jira.mongodb.org/browse/DOCSP-33079)

[BUILD](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=650de451492df295cfee55a7)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -14,6 +14,48 @@
...
 
+.. _atlas-cli-1.12.0:
+
+{+atlas-cli+} 1.12.0
+--------------------
+
+- Drops support for Ubuntu 18.04.
+- Adds :ref:`atlas-deployments` commands so that you can work with
+  |service|, including using |fts| and {+avs+}, throughout the entire 
+  software development lifecycle, from your local environment to the
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt]:
```suggestion
  software development lifecycle from your local environment to the 
```

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -14,6 +14,48 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.12.0:
+
+{+atlas-cli+} 1.12.0
+--------------------
+
+- Drops support for Ubuntu 18.04.
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:22]: Suggestion: With such big announcements this release, maybe make this last?
```suggestion
- Drops support for Ubuntu 18.04.
```

File: source/atlas-cli-changelog.txt [hunk 3]
```diff
@@ -14,6 +14,48 @@
...
+- Adds :ref:`atlas-deployments` commands so that you can work with
+  |service|, including using |fts| and {+avs+}, throughout the entire 
+  software development lifecycle, from your local environment to the 
+  cloud. To learn more, see :ref:`atlas-cli-local-cloud`.
+- Adds the following new commands to manage {+atlas-sp+} instances
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:29]: Suggestion: link to the Streams Processing page for more info if they're unfamiliar with it

**Review by colm-quinn — APPROVED** [review]
Thank you, LGTM!

File: source/atlas-cli-changelog.txt [hunk 4]
```diff
@@ -14,6 +14,48 @@
...
+  and connection registries:
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:30]: Question: are these the first streams processing commands or were there other ones released before this? If they're the first consider just linking to the top-level streams processing command to align with what you did with atlas deployments (or, conversely, list out the deployments commands). This is optional though, not blocking

File: source/atlas-cli-changelog.txt [hunk 5]
```diff
@@ -14,6 +14,48 @@
...
+
+{+atlas-cli+} 1.11.0
+--------------------
+
+- Fixes an issue that occurred when you created x509 certs for database 
+  users. The command didn't output the certificate. If you depend on 
+  the previously incorrect output, this update is a breaking change.
+
+- :ref:`atlas-alerts-settings` create and update commands perform 
+  validation that helps you formulate all required arguments per each
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:51]: Nit: "for each" is more commonly used than "per each" so will improve reader comprehension
```suggestion
  validation that helps you formulate all required arguments for each 
```

**Review by blva — APPROVED** [review]

File: source/atlas-cli-changelog.txt [hunk 6]
```diff
@@ -14,6 +14,48 @@
...
+  the previously incorrect output, this update is a breaking change.
+
+- :ref:`atlas-alerts-settings` create and update commands perform 
+  validation that helps you formulate all required arguments per each 
+  alert type.
+
+- Fixes a bug where the :ref:`atlas-projects-list` command ignored the 
+  ``--orgId`` flag.
+
+- Fixes an issue with running cloud provider access commands.
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:57]:
```suggestion
- Fixes an issue that occurred while running cloud provider access commands.
```

File: source/atlas-cli-changelog.txt [hunk 7]
```diff
@@ -14,6 +14,48 @@
...
+
+- Supports configuring alerts with a file.
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:59]:
```suggestion
- Supports configuring alerts with a file. To learn more,
  see :doc:`</reference/json/alert-config-file>.
```

**Review by sarahsimpers — APPROVED** [review]
LGTM with some non-blocking suggestions, thank you!

**Pull request merged**